### DISCUSSION TITLE
Unbinding ctrl q

### DISCUSSION THREAD

**Discussion by aral** [zellij-org/zellij#2159] [Q&A]
Hey folks,

In my config, if I do:

```kdl
keybinds {
  unbind "Ctrl q"

  …

  shared_except "locked" {
    bind "Ctrl x" { Quit; }
  }
}
```

Ctrl+q _does not_ unbind and still works.

However, if I do:

```kdl
keybinds {
  normal {
    unbind "Ctrl q"
  }

  …

  shared_except "locked" {
    bind "Ctrl x" { Quit; }
  }
}
```

Then Ctrl+q is correctly unbound.

Is this expected behaviour or a bug? (I thought unbinding globally would do so for all modes.)

_I use Ctrl+q to exit [Helix Editor](https://helix-editor.com) so I keep exiting Zellij by mistake, which is why I’m unbinding it and binding it to the otherwise unused Ctrl+x._

**Comment by imsnif** [comment]
Hey, good find! This is definitely a bug. As an additional workaround, you could also specify `keybinds clear-defaults=true`, which would unbind everything not explicitly in the file (but then you might have to add stuff manually when new versions bring in new capabilities).

Would you be willing to open an issue for this so that we can keep track more easily?

**Reply by aral** [reply]
Done :)
